web-console/future-openshift/cron-jobs/cron-jobs.md

```diff
@@ -0,0 +1,27 @@
+# Cron Jobs
+
+## Create Cron Job
+![](img/cron-job-list.png)
+- Clicking the `Create Cron Job` button launches the creation form.
+
+![](img/create-cron-job.png)
+- Users are prompted to enter a name for the Cron Job, a schedule, and a job template to run.
```

Comment:
@bparees already mentioned that during call, but I'll repeat, it would be nice if this asks you for your image name, optionally for a command, and filled in the rest for you. As an example from CLI we do:
```
 oc create cronjob NAME --image=image --schedule='0/5 * * * ?' -- [COMMAND] [args...] [flags]
```
where a user needs to provide the name, schedule and image these 3 are required, then optionally he can provide the command and restart policy. That last one can but doesn't have to be added.

Reply:
With a fallback to advanced screen where you'd have yaml editor to write that down.